test/core/miner/CMakeLists.txt

```diff
@@ -35,6 +35,7 @@ addtest(tipset_cache_test
     )
 target_link_libraries(tipset_cache_test
     tipset_cache
+    miner_types
```

Comment:
Does it need this?

Reply:
Yes, because of deadlines

Reply:
do you mean move it to tipset_cahce.hpp ?